domehunter/gRPC-server/hx2dome.proto_server.py

```diff
@@ -571,7 +573,25 @@ def serve(kwargs_dict):
                         to the TheSkyX.")
     parser.set_defaults(server_testing=False)
 
-    kwargs_dict = vars(parser.parse_args())
+    parser.add_argument('-y', '--yamlconfig',
+                        dest='yamlconfig',
+                        help="YAML file containing cofiguration details for \
+                        the dome controller.")
+    default_config = os.path.join(os.path.abspath(os.path.dirname(__file__)),
+                                  'dome_controller_config.yml')
+    parser.set_defaults(yamlconfig=default_config)
+
+    args = parser.parse_args()
+
+    # TODO: have some way of running dome with the defaults defined in the init
+    # rather than running from a specified or default yaml file
```

Comment:
Not sure I understand this todo. Should just specify default values on the keyword arguments in the init.

Reply:
what I mean is having a way to run the server using the defaults from the dome class. At the moment you can specify a yaml config and if you dont it just uses a default yaml config and passes that through to the dome instance. 
I just thought there might be a scenario where you only want to pass in just `home_azimuth` and adopt the defaults from the dome class definition (as opposed to using the default yaml config).